src/Jackett.Common/Definitions/flood.yml

```diff
@@ -0,0 +1,123 @@
+---
+id: flood
+name: Flood
+description: "Flood is a Private Torrent Tracker for MOVIES / TV"
+language: en-US
+type: private
+encoding: UTF-8
+links:
+  - https://flood.st/
+
+caps:
+  categorymappings:
+    - {id: movie, cat: Movies, desc: "Movies"}
+    - {id: show, cat: TV, desc: "TV"}
+
+  modes:
+    search: [q]
+    tv-search: [q, season, ep, tmdbid, imdbid]
+    movie-search: [q, tmdbid, imdbid]
+
+settings:
+  - name: apikey
+    type: text
+    label: API key
+  - name: info_key
+    type: info
+    label: About your API key
+    default: "Find or generate a new API key by accessing the <b>Security</b> tab of your <a href=\"https://flood.st/settings/security\" target=\"_blank\">account settings</a>."
+  - name: order_by
+    type: select
+    label: Sort requested from site
+    default: id
+    options:
+      id: created
+      name: title
+      size: size
+      seeders: seeders
+      leechers: leechers
+      times_completed: grabs
+      release_date: release date
+  - name: order_dir
+    type: select
+    label: Order requested from site
+    default: desc
+    options:
+      desc: desc
+      asc: asc
+
+login:
+  path: api/test
+  method: get
+
+search:
+  paths:
+    - path: api/torrents
+      response:
+        type: json
+
+  headers:
+    Authorization: ["Bearer {{ .Config.apikey }}"]
+
+  inputs:
+    name: "{{ .Keywords }}"
+    show_season_number: "{{ .Query.Season }}"
+    show_episode_number: "{{ .Query.Ep }}"
+    tmdb_id: "{{ .Query.TMDBID }}"
+    imdb_id: "{{ .Query.IMDBID }}"
+    order_by: "{{ .Config.order_by }}"
+    order_dir: "{{ .Config.order_dir }}"
+    limit: 100
+
+  keywordsfilters:
+    - name: re_replace
+      args: ["\\.", " "]
+
+  rows:
+    selector: items
+    count:
+      selector: total
+
+  fields:
+    _id:
+      selector: id
+    title:
+      selector: name
+    infohash:
+      selector: info_hash
+    category:
+      selector: media_type
+      filters:
+        - name: replace
+          args: ["show_season", "show"]
+        - name: replace
+          args: ["show_episode", "show"]
```

Comment:
Is there a reason to do this, rather than just having 3 categories (i.e. `Movie`, `Show Season`, and `Show Episode`)?